lib/server.js

```diff
@@ -524,10 +522,13 @@ function createServer(options) {
                 var size_histogram = collector.getCollector(
                     METRIC_SIZE_HISTOGRAM);
```

Comment:
Line 516 - 523 could be in the `if` block.

Reply:
Indeed. Thanks for the review. Applied the suggested changes and pushed again.